IllegalArgumentException in NotifcationService.onStartCommand()

Crash reported via Google Play developer console.

**Application version** 0.9.2
**Android version** Android 6.0
**Device** Galaxy S6 Edge (zerolteatt)

```
java.lang.RuntimeException: Unable to start service com.mapzen.erasermap.service.NotificationService@8328582 with null: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter intent
	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4080)
	at android.app.ActivityThread.access$2400(ActivityThread.java:222)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1898)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:158)
	at android.app.ActivityThread.main(ActivityThread.java:7229)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
Caused by: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter intent
	at com.mapzen.erasermap.service.NotificationService.onStartCommand(NotificationService.kt)
	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4063)
	... 8 more
```

--- a/app/src/main/kotlin/com/mapzen/erasermap/service/NotificationService.kt
+++ b/app/src/main/kotlin/com/mapzen/erasermap/service/NotificationService.kt
@@ -13,7 +13,7 @@ class NotificationService : Service() {
 
     internal var binder: IBinder = NotificationBinder(this)
 
-    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
+    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
         return Service.START_STICKY
     }
 

